client/signup/steps/about/index.jsx

```diff
@@ -443,57 +489,82 @@ class AboutStep extends Component {
 		);
 	}
 
+	scrollUp() {
+		// Didn't use setInterval in order to fix delayed scroll
+		while ( window.pageYOffset > 0 ) {
+			window.scrollBy( 0, -10 );
+		}
+	}
+
 	renderContent() {
 		const { translate, siteTitle } = this.props;
 
+		const pressableWrapperClassName = classNames( 'about__pressable-wrapper', {
+			'about__wrapper-is-hidden': ! this.state.showStore,
+		} );
+
+		const aboutFormClassName = classNames( 'about__form-wrapper', {
+			'about__wrapper-is-hidden': this.state.showStore,
+		} );
+
 		return (
 			<div className="about__wrapper">
-				<form onSubmit={ this.handleSubmit }>
-					<Card>
-						<FormFieldset>
-							<FormLabel htmlFor="siteTitle">What would you like to name your site?</FormLabel>
-							<FormTextInput
-								id="siteTitle"
-								name="siteTitle"
-								placeholder="eg: Mel's Diner, Stevie’s Blog, Vail Renovations"
-								defaultValue={ siteTitle }
-								onChange={ this.handleChangeEvent }
-							/>
-						</FormFieldset>
-
-						<FormFieldset>
-							<FormLabel>What will your site be about?</FormLabel>
-							<FormTextInput
-								id="siteTopic"
-								name="siteTopic"
-								placeholder="eg: Fashion, travel, design, plumber, electrician"
-								value={ this.state.siteTopicValue }
-								onChange={ this.handleSuggestionChangeEvent }
-								onBlur={ this.hideSuggestions }
-								onKeyDown={ this.handleSuggestionKeyDown }
-								autoComplete="off"
-							/>
-							<Suggestions
-								ref={ this.setSuggestionsRef }
-								query={ this.state.query }
-								suggestions={ this.getSuggestions() }
-								suggest={ this.handleSuggestionMouseDown }
-							/>
-						</FormFieldset>
-
-						<FormFieldset>
-							<FormLabel>What’s the primary goal you have for your site?</FormLabel>
-							{ this.renderGoalCheckboxes() }
-						</FormFieldset>
-
-						{ this.renderExperienceOptions() }
-					</Card>
-					<div className="about__submit-wrapper">
-						<Button primary={ true } type="submit">
-							{ translate( 'Continue' ) }
-						</Button>
-					</div>
-				</form>
+				<div className={ pressableWrapperClassName }>
+					<PressableStoreStep
+						{ ...this.props }
+						onBackClick={ this.handleStoreBackClick }
+						setRef={ this.setPressableStore }
+					/>
+				</div>
+
+				<div className={ aboutFormClassName }>
+					<form onSubmit={ this.handleSubmit }>
+						<Card>
+							<FormFieldset>
+								<FormLabel htmlFor="siteTitle">What would you like to name your site?</FormLabel>
```

Comment:
Please `translate`

Reply:
Thanks @allendav. This screen is still hidden behind an a/b test which means it will only be shown to English visitors. Since the test passed, I'm in the process of translating all the strings in another PR.